offscreen_effect_player/offscreen_effect_player.cpp

```diff
@@ -24,6 +24,7 @@ namespace bnb::oep
             m_ort->init(width, height);
             m_ort->activate_context();
             m_ep->surface_created(width, height);
+            m_ep->set_render_surface(m_ort->get_layer());
```

Comment:
This also should be under define

Reply:
Check style - as far as I remember Banuba SDK assumes #ifdef at the beginning of the line, without indentation.

Reply:
Again the implementation of Metal is in the sample, so this specific should be moved to the sample.